packages/binding-http/tsconfig.json

```diff
@@ -2,7 +2,8 @@
     "extends": "../../tsconfig.json",
     "compilerOptions": {
         "outDir": "dist",
-        "rootDir": "src"
+        "rootDir": "src",
+        "strict": true
```

Comment:
Do we want to add 

`"strictFunctionTypes": true`

as we do in the other tsconfigs?

Reply:
I saw the comment in https://github.com/eclipse-thingweb/node-wot/pull/1047#discussion_r1281537969
Hence we can tackle this comment as addressed